### DISCUSSION TITLE
Cannot update ”AgentModule”

### DISCUSSION THREAD

**Discussion by skysky0208** [yasserfarouk/scml#109] [General]
I tried to update "Agent Class" and "Agent Module", but "Agent Module" is grayed out and cannot be filled in.

Please confirm this problem.

AgentID：20102
AgentName：SkyAgent

<img width="791" alt="20220622" src="https://user-images.githubusercontent.com/55139437/174989296-bd0cc0b4-f123-473a-8bf4-8fc8034155a6.png">

**Comment by yasserfarouk** [comment]
Fixed. Please try again and let me know if you are still having trouble.

sorry for the inconvenience
yasser


On June 22, 2022 at 18:04:54, SakamotoNatsumi ***@***.***)
wrote:

I tried to update "Agent Class" and "Agent Module", but "Agent Module" is
grayed out and cannot be filled in.

Please confirm this problem.

AgentID：20102
AgentName：SkyAgent

[image: 20220622]
<https://user-images.githubusercontent.com/55139437/174989296-bd0cc0b4-f123-473a-8bf4-8fc8034155a6.png>

—
Reply to this email directly, view it on GitHub
<https://github.com/yasserfarouk/scml/discussions/109>, or unsubscribe
<https://github.com/notifications/unsubscribe-auth/AARYQB3TQ5HCC6EN3FGF5OTVQLJLLANCNFSM5ZPOFRSA>
.
You are receiving this because you are subscribed to this thread.Message
ID: ***@***.***>

**Reply by skysky0208** [reply]
Thenk you for resolving the issue.
